boards/PDM/Src/Io/Io_Efuse.c

```diff
@@ -0,0 +1,935 @@
+#include <assert.h>
+#include "Io_Efuse.h"
+#include "Io_CurrentSense.h"
+
+#pragma GCC diagnostic ignored "-Wconversion"
+
+#define EXIT_OK_OR_RETURN(CODE) \
```

Comment:
I think this should be moved into App_SharedExitCode.h

Reply:
MOVED

Reply:
Should we also do (code != EXIT_CODE_OK) for the condition just to be extra clear

Reply:
YAR